docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  leantime_web:
+    image: leantime/leantime:latest
+    restart: always
+    environment:
+      - DB_HOST=mysql_leantime
+      - MYSQL_USER=admin
+      - MYSQL_PASSWORD=321.qwerty
+      - MYSQL_DATABASE=leantime
```

I would suggest environment variables which can then savely be substitued through an .env file like in my setup: 
https://github.com/MelvinLoos/leantime-docker-compose
```suggestion
      - MYSQL_USER=${DB_USER}
      - MYSQL_PASSWORD=${DB_PASSWORD}
      - MYSQL_DATABASE=${DB_DATABASE}
```